apiary-metastore-metrics/src/main/java/com/expedia/apiary/extensions/metastore/metrics/CloudwatchReporter.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (C) 2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.apiary.extensions.metastore.metrics;
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Gauge;
+
+import java.io.Closeable;
+import java.util.TimerTask;
+import java.util.List;
+import java.util.Map;
+
+import com.amazonaws.services.cloudwatch.AmazonCloudWatch;
+import com.amazonaws.services.cloudwatch.AmazonCloudWatchClientBuilder;
+import com.amazonaws.services.cloudwatch.model.MetricDatum;
+import com.amazonaws.services.cloudwatch.model.PutMetricDataRequest;
+import com.amazonaws.services.cloudwatch.model.PutMetricDataResult;
+import com.amazonaws.services.cloudwatch.model.StandardUnit;
+import com.amazonaws.services.cloudwatch.model.Dimension;
+
+/**
+ * TODO: need to update this to implement CodahaleReporter when migrating to hive 3.0.0, 
+ * similar to https://github.com/apache/hive/blob/master/common/src/java/org/apache/hadoop/hive/common/metrics/metrics2/JsonFileMetricsReporter.java
+ */
+
+class CloudwatchReporter implements Closeable {
+    private java.util.Timer timer = null;
+    private AmazonCloudWatch cw = null;
+    private MetricRegistry metricRegistry;
+
+    public CloudwatchReporter(MetricRegistry metricRegistry) {
+        this.metricRegistry=metricRegistry;
+    }
+
+    public void start() {
+      this.cw = AmazonCloudWatchClientBuilder.defaultClient();
```

Comment:
If start() is called more than once it will overwrite these values, not sure what kind of side effects this will have, I'm guessing an orphaned timer object. Perhaps it would be better to move lines 49 and 50 into the constructor?